.npmrc

```diff
@@ -0,0 +1 @@
+side-effects-cache=false
```

Comment:
we should probably merge https://github.com/svelte-add/svelte-add/pull/385 first and see if it allows us to remove this file

Reply:
After merging #385 and removing this file the pipeline is still working. Thanks for this note.